R/whatWQPdata.R

```diff
@@ -31,19 +32,30 @@ whatWQPsamples <- function(...,
   }
 
   values <- sapply(values, function(x) utils::URLencode(x, reserved = TRUE))
-
-  baseURL <- drURL("ActivityWQX3", arg.list = values)
-  wqp_message_now("ActivityWQX3")
+  
+  if(legacy){
+    baseURL <- drURL("Activity", arg.list = values)
+  } else {
+    baseURL <- drURL("ActivityWQX3", arg.list = values)
+  }
```

Comment:
So in the development version, this function only performed WQX3.0 calls, but this version adds in the choice?